UnityProject/Assets/Editor/CompileMaterialWindow.cs

```diff
@@ -17,7 +17,7 @@ public static void ShowWindow()
 
     private void OnGUI()
     {
-        GUILayout.Label("Walls", EditorStyles.boldLabel);
+        GUILayout.Label("Pixie", EditorStyles.boldLabel);
```

Comment:
This should remain "Walls" as it's just the header in the Unity editor menu. You probably meant to change line 33 back

Reply:
My b, whoops-